### DISCUSSION TITLE
Loading is being performed inadequately.

### DISCUSSION THREAD

**Discussion by lucioerlan** [community/community#47205] [Other Feature Feedback, Questions, & Ideas]
### Bug

This is a request to investigate the general page loading issue. It appears that the loading spinner is being activated inappropriately. I have encountered instances while browsing the site where the spinner continues to spin endlessly.

This issue started to occur for me approximately **two days** ago.

evidences:
![spinner](https://user-images.githubusercontent.com/67064886/218374708-eec72833-cba2-4ca4-8c31-81c07e2eeaa2.gif)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @lucioerlan ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
